src/elasticsearch/dataservices.py

```diff
@@ -0,0 +1,67 @@
+import asyncio
+import logging
+
+from src.elasticsearch.queries import DATASERVICE_AGGREGATION_FIELDS
+from src.elasticsearch.utils import elasticsearch_ingest, add_org_paths_to_document, EsMappings, \
+    get_all_organizations_with_publisher
+from src.rdf_namespaces import JSON_RDF
+from src.utils import ContentKeys
+from src.service_requests import fetch_dataservices_from_dataservice_harvester, fetch_publishers_from_dataset_harvester
+from src.utils import FetchFromServiceException, ServiceKey
+
+
+def insert_dataservices(success_status, failed_status):
+    try:
+        loop = asyncio.get_event_loop()
+    except RuntimeError:
+        loop = asyncio.new_event_loop()
+        asyncio.set_event_loop(loop)
+
+    try:
+        collection_tasks = asyncio.gather(
+            fetch_dataservices_from_dataservice_harvester(),
+            fetch_publishers_from_dataset_harvester()
```

Comment:
Hent ur organisasjoner for dataservices i stedet?

Reply:
Endrer og refaktorerer "build_dataset_publisher_query" til å hete "build_publisher_from_graph_query" og ta "from" graf som input. Gjenbruker da samme build for dataservices.

Reply:
Rettet